WcaOnRails/config/locales/en.yml

```diff
@@ -1044,6 +1056,44 @@ en:
     post_announcement: "Post competition announcement"
     show:
       general_info: "General info"
+      schedule: "Schedule"
+    #context: competition's schedule tab
+    schedule:
+      #context: when selecting the kind of schedule to display
+      display_as:
+        label: "Display the schedule as:"
+        table: "Table"
+        calendar: "Calendar"
+      #context: when selecting the rooms to display on the schedule
+      display_for_room: "Display schedule for:"
+      #context: explanation of what time limit and cutoff are
+      time_limit_information:
+        time_limit_html: "If you reach the time limit during your solve, the judge will stop you and your result will be DNF (see %{regulation_link})."
+        #context: as used in "please see Regulation A1a4"
+        regulation_link_text: "Regulation %{number}"
+        #context: as used in "please see Guideline A1a2++"
+        guideline_link_text: "Guideline %{number}"
+        #context: used in the sentence "a cumulative time limit may be enforced [...]"
+        cumulative_time_limit: "cumulative time limit"
+        cumulative_one_round_html: "A %{cumulative_time_limit} may be enforced (see %{regulation_link})."
+        cumulative_across_rounds_html: "A %{cumulative_time_limit} may be enforced across rounds (see %{guideline_link})."
+        cutoff_html: "The ranking or time to meet to proceed to the second phase of a combined round (see %{regulation_link})."
```

Comment:
Nitpick: It should be "result", and "result" only - not "ranking or result", and I think it would be better to use "beat" instead of "meet", to make it clear that it has to be strictly better.

Reply:
You're definitely right, I took it from the current "show events" page that was introduced in #1856 ;)